.travis.yml

```diff
@@ -1,5 +1,11 @@
 language: node_js
 node_js:
-  - "10"
+  - "12"
 notifications:
   email: false
+script:
+  - npm run test
+  - npm run lint
+branches:
+  only: 
+    - master
```

Comment:
will this stop travis runing on pull requests?

Reply:
From what I remember no - there is a separete flag somewhere in travis which controls "pull requests to master", but let me double check, maybe I'm wrong

Reply:
re-checked - does not block PRs - controlled by this:
![2019-09-17_00h29_42](https://user-images.githubusercontent.com/15635498/64995390-c2452d00-d8e3-11e9-8994-e8195654be94.png)